### PR TITLE
[ML] Increase Model Download Timeout In YAML Tests

### LINKED ISSUE #116899: [CI] InferenceRestIT class failing
**Build Scans:**
- [elasticsearch-periodic-platform-support #4879 / debian-11_platform-support-unix](https://gradle-enterprise.elastic.co/s/iqqd3634ksoqo)
- [elasticsearch-periodic #4879 / concurrent-search-tests](https://gradle-enterprise.elastic.co/s/w6cunhz2iuo4e)
- [elasticsearch-periodic-platform-support #4854 / ubuntu-1804_platform-support-unix](https://gradle-enterprise.elastic.co/s/v4zve7isnmggs)
- [elasticsearch-pull-request #41407 / part-2](https://gradle-enterprise.elastic.co/s/5jynh4ecezywa)
- [elasticsearch-pull-request #41407 / part-2](https://gradle-enterprise.elastic.co/s/rb7zekrofcs2o)
- [elasticsearch-pull-request #41359 / part-2](https://gradle-enterprise.elastic.co/s/ndjfn6qudsxgs)
- [elasticsearch-pull-request #41211 / part-2](https://gradle-enterprise.elastic.co/s/bslmqlzrllau4)
- [elasticsearch-pull-request #41211 / part-2](https://gradle-enterprise.elastic.co/s/rx37jqskeqe62)
- [elasticsearch-intake #12842 / part2](https://gradle-enterprise.elastic.co/s/ttuob2hffvoly)
- [elasticsearch-pull-request #41210 / part-2](https://gradle-enterprise.elastic.co/s/472y3437zkqnw)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:inference:yamlRestTest" --tests "org.elasticsearch.xpack.inference.InferenceRestIT.test {p0=inference/40_semantic_text_query/Query a field that uses the default ELSER 2 endpoint}" -Dtests.seed=A2266C56205315AD -Dtests.locale=en-FM -Dtests.timezone=Europe/Simferopol -Druntime.java=23
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.inference.InferenceRestIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(org.elasticsearch.xpack.inference.InferenceRestIT),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.RuntimeException: Failure at [inference/40_semantic_text_query:862]: 120,000 milliseconds timeout on connection http-outgoing-1 [ACTIVE]
```

**Issue Reasons:**
- [main] 13 failures in class org.elasticsearch.xpack.inference.InferenceRestIT (2.4% fail rate in 542 executions)
- [main] 8 failures in step part-2 (4.0% fail rate in 198 executions)
- [main] 2 failures in step part2 (2.9% fail rate in 68 executions)
- [main] 2 failures in pipeline elasticsearch-periodic-platform-support (16.7% fail rate in 12 executions)
- [main] 6 failures in pipeline elasticsearch-pull-request (3.1% fail rate in 194 executions)
- [main] 2 failures in pipeline elasticsearch-intake (2.9% fail rate in 68 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

### PR DESCRIPTION
**Pull request by Mikep86**
Increase the timeout used in inference YAML tests for model download. Fixes #116899.

### CODE REVIEW

File: muted-tests.yml [hunk 1]
```diff
@@ -231,14 +231,9 @@ tests:
 - class: org.elasticsearch.search.basic.SearchWithRandomIOExceptionsIT
   method: testRandomDirectoryIOExceptions
   issue: https://github.com/elastic/elasticsearch/issues/114824
-- class: org.elasticsearch.xpack.inference.InferenceRestIT
-  method: test {p0=inference/30_semantic_text_inference/Calculates embeddings using the default ELSER 2 endpoint}
-  issue: https://github.com/elastic/elasticsearch/issues/116542
```
Comment (Mikep86) [inline muted-tests.yml:236]: Also re-enable this test, which was fixed by #116749

**Review by kderusso — APPROVED** [review]

**Comment by kderusso** [comment]
Approved pending green CI :)

**Review by davidkyle — APPROVED** [review]
LGTM

**Comment by dnhatn** [comment]
I've merged this PR to unblock an ongoing release.

**Pull request merged**